static/src/javascripts/projects/common/modules/discussion/activity-stream.js

```diff
@@ -0,0 +1,146 @@
+// @flow
+
+import bonzo from 'bonzo';
+import bean from 'bean';
+import $ from 'lib/$';
+import {
+    getUrlVars,
+    constructQuery,
+    replaceQueryString,
+    pushQueryString,
+} from 'lib/url';
+import Component from 'common/modules/component';
+import discussionApi from 'common/modules/discussion/api';
+
+declare class PopStateEvent extends Event {
```

Comment:
Adding `declare` to this Flow declaration fixes issue previously raised after https://github.com/guardian/frontend/pull/17549 was merged which affected Safari users.